### PR TITLE
chore(execution): close unified constitution cutover

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Completes the unified execution constitution cutover across phases 0-7 and adds the final packet-grade closeout evidence. It lands the constitutional kernel, run-bound execution model, proof and disclosure planes, adapter/support-target governance, and build-to-delete closeout surfaces needed for the final target-state claim.

## Why

This closes the remaining packet work so the repository can retain one final verdict for the unified execution constitution target state and carry promotion/archive readiness as durable evidence.

No-Issue: repository governance cutover and closeout

## How

Implements the phase bundles as durable repo surfaces, deletes persona and label-native scaffolding from the live path, adds adapter/support-target and build-to-delete governance contracts, and records the final cutover verdict. The active packet manifests were also normalized to the current proposal-validator contract so promotion and archive readiness are validator-backed rather than narrative-only.

## Profile Selection Receipt

- Semantic version source(s): `version.txt`, `/.octon/octon.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): repo-local governance/runtime/docs changes; medium downtime tolerance; no external coordination required; seeded evidence backfill required; rollback is full revert of this branch; blast radius is high because contracts, runtime artifacts, CI, and closeout evidence move together; constitutional kernel, fail-closed routing, and disclosure requirements remain mandatory
- Tie-break status: `atomic` selected because the cutover only holds if contracts, runtime artifacts, CI, and closeout evidence agree in one branch
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): `n/a`

## Implementation Plan

- Workstreams and scope: phases 0-7 of the cutover plus the final packet-grade closeout and promotion-readiness validation
- Public interfaces/types/contracts affected: constitutional contract families, runtime manifests, support-targets, adapter manifests, build-to-delete governance contracts, proposal packet manifests
- Test scenarios: final closeout validator suite, phase validators, run/disclosure validators, proposal validators, `cargo check -p octon_kernel`
- Assumptions/defaults: `main` is the base branch; packet archive remains a separate lifecycle step after this PR lands

## Impact Map (code, tests, docs, contracts)

- Code: GitHub workflows, engine runtime authorization/workflow paths, orchestration run writers, evaluator adapter runner
- Tests: runtime/assurance/governance validators and targeted kernel checks
- Docs: ingress, agency/runtime docs, governance docs, packet docs, closeout evidence
- Contracts: objective, authority, runtime, assurance, disclosure, adapters, retention, support-target, retirement, and closeout contracts

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

- This is a large branch because it closes the full cutover plus final closeout in one reviewable unit.
- Packet promotion/archive is intentionally left as a separate lifecycle operation after merge.
- Local Cargo build cache remains uncommitted and out of scope for the PR.

## Testing

- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase7-build-to-delete-institutionalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-constitution-closeout.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh`
- `bash .octon/framework/orchestration/runtime/runs/_ops/scripts/validate-runs.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase6-simplification-deletion.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase5-adapter-support-target-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase4-proof-lab-enforcement.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-lifecycle-normalization.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-objective-binding-cutover.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-governance.sh`
- `bash .octon/framework/agency/_ops/scripts/validate/validate-agency.sh`
- `bash .octon/framework/agency/_ops/scripts/validate/validate-autonomy-labels.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-bootstrap-ingress.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-harness-structure.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-proposal-standard.sh --package .octon/inputs/exploratory/proposals/architecture/octon-unified-execution-constitution-cutover`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-architecture-proposal.sh --package .octon/inputs/exploratory/proposals/architecture/octon-unified-execution-constitution-cutover`
- `cargo check --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel`
- `git diff --check`

## Rollout

n/a

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert the cutover branch commit
- Rollback handle: `git revert c8f235ccf`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: added or updated runtime, adapter, assurance, retention, lab, and disclosure schemas
- Threat model update/link: host-native projections remain non-authoritative; authority stays artifact-routed; unsupported tuples remain fail-closed; disclosure remains subordinate to authority and retained evidence

## Observability and Performance

- Traces/logs/metrics for changed flows: run evidence, replay pointers, run/harness cards, build-to-delete review packet, and authority control receipts
- Representative traces for high risk changes: seeded `run-wave3-runtime-bridge-20260327` and `run-wave4-benchmark-evaluator-20260327` evidence bundles
- Performance or bundle impact: no claimed product-runtime regression; change is primarily governance/runtime-contract/validator surface

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: refreshed `/.octon/generated/cognition/summaries/decisions.md` via `sync-runtime-artifacts.sh`

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] All review conversations resolved

## Screenshots / Notes

No UI screenshots. Final closeout evidence says the packet is ready for promotion and archive as a separate follow-up lifecycle step after merge.

### CODE REVIEW

File: .octon/framework/orchestration/runtime/_ops/scripts/write-run.sh [hunk 1]
```diff
@@ -1624,6 +1975,7 @@ PY
           + (if $queue_item_id != "" then {queue_item_id:$queue_item_id} else {} end)
           + (if $parent_run_id != "" then {parent_run_id:$parent_run_id} else {} end)
           + (if $coordination_key != "" then {coordination_key:$coordination_key} else {} end)
+          + (if $status == "succeeded" or $status == "failed" or $status == "cancelled" then {external_replay_index_path:$external_replay_index_path} else {} end)
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/orchestration/runtime/_ops/scripts/write-run.sh:1979]: Updated `write-run.sh` so terminal completion persists `external_replay_index_path` after `update_run_contract_status` refreshes the retained evidence bundle. I also fixed the evidence-write ordering so `retained-run-evidence.yml` exists before the proof suites run, and added regression coverage in `test-shared-runtime-primitives.sh` for high-tier completion pointers.

File: .github/workflows/ai-review-gate.yml [hunk 2]
```diff
@@ -237,28 +223,40 @@ jobs:
...
-
-          gh pr edit "${GH_PR_NUMBER}" --repo "${GH_REPO}" --add-label "ai-gate:required" >/dev/null 2>&1 || true
+          status="granted"
 
-          if [[ "${blockers}" -gt 0 && "${waived}" != "true" ]]; then
-            gh pr edit "${GH_PR_NUMBER}" --repo "${GH_REPO}" --add-label "ai-gate:blocker" >/dev/null 2>&1 || true
-          else
-            gh pr edit "${GH_PR_NUMBER}" --repo "${GH_REPO}" --remove-label "ai-gate:blocker" >/dev/null 2>&1 || true
+          if [[ "${blockers}" -gt 0 ]]; then
+            status="staged"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .github/workflows/ai-review-gate.yml:244]: Updated the AI gate workflow to derive the canonical approval artifact status from `gate_pass` first, so enforced failures such as provider-unavailable are emitted as `denied` instead of `granted`. I also added a denied-projection regression check in `test-authority-control-tooling.sh`.
